### DISCUSSION TITLE
Keycloak.X and syslog configuration

### DISCUSSION THREAD

**Discussion by cgeorgilakis** [keycloak/keycloak#17691] [Archive]
We use version 18 in Quarkus (Keycloak.X).
We have noticed that in all installation by default Keycloak write logs in syslog with _app-name_ as _bash_ . This is not ideal but not big problem.

The problem is that in one installation, we want to configure Keycloak with syslog. Especially, we want to change _app-name_ to _keycloak_.
We use configuration in [syslog logging of Quarkus](https://quarkus.io/guides/logging#quarkus-log-logging-log-config_quarkus.log.syslog-syslog-logging). We put this configuration (fe quarkus.log.syslog.app-name=keycloak ) in quarkus.properties file as mentioned in [Keycloak configuration](https://www.keycloak.org/server/configuration). However, no change in syslog logging is done.

How can we configure syslog with Keycloak X (Quarkus)?

**Comment by dasniko** [comment]
I have this working properly, like discussed in #10201, since KC 19 (or 20? don't know exactly).
Might be worth upgrading your environment.

**Comment by darius-m** [comment]
If all you need is changing the prefix, can set `SyslogIdentifier=keycloak` in the `Service` section of your systemd config file, similar to what is mentioned in [this serverfault answer](https://serverfault.com/a/909973).

**Reply by cgeorgilakis** [reply]
Thanks you @darius-m.
This work. It is enough for us.
Unfortunately we can not migrate to newer version.
